src/dnx/pal.h

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-void GetNativeBootstrapperDirectory(LPTSTR szPath);
+LPTSTR GetNativeBootstrapperDirectory();
```

Comment:
Why this can't just return std::string? It would save all the hassle with cleaning up etc. I have started working on cleaning LPCTSTR/LPTSTR/LPSTR mess by introducing cross platform types definitions (WIP: https://github.com/aspnet/dnx/commit/cfe5955281c87825877842f8bc21c068a2203671#diff-f83d3802c96eed0aa99ee2b81e85e9ffR9). Using stl would also make it much easier to do string operations.